### PR TITLE
chore: typos & refine comments

### PR DESCRIPTION
**Pull request by ben1009**
- fix  typos by `typos -w`, suggest add [typos-cli](https://github.com/crate-ci/typos) to gha
- refine comment

### CODE REVIEW

File: mini-lsm-mvcc/src/block/iterator.rs [hunk 1]
```diff
@@ -15,7 +15,7 @@ pub struct BlockIterator {
     block: Arc<Block>,
     /// the current key at the iterator position
     key: KeyVec,
-    /// the value range from the block
+    /// the current value range in the block.data, coresponds to the current key
```
Comment (xzhseh) [inline mini-lsm-mvcc/src/block/iterator.rs:18]:
```suggestion
    /// the current value range in the block.data, corresponds to the current key
```

File: mini-lsm-starter/src/block/iterator.rs [hunk 2]
```diff
@@ -13,7 +13,7 @@ pub struct BlockIterator {
     block: Arc<Block>,
     /// The current key, empty represents the iterator is invalid
     key: KeyVec,
-    /// the value range from the block
+    /// the current value range in the block.data, coresponds to the current key
```
Comment (xzhseh) [inline mini-lsm-starter/src/block/iterator.rs:16]: ditto

File: mini-lsm/src/block/iterator.rs [hunk 3]
```diff
@@ -15,7 +15,7 @@ pub struct BlockIterator {
     block: Arc<Block>,
     /// the current key at the iterator position
     key: KeyVec,
-    /// the value range from the block
+    /// the current value range in the block.data, coresponds to the current key
```
Comment (xzhseh) [inline mini-lsm/src/block/iterator.rs:18]: ditto

**Review by xzhseh — COMMENTED** [review]
and for the pr title, `chroe` -> `chore`. 😁

**Review by skyzh — APPROVED** [review]

**Pull request merged**